src/main/java/com/example/nlern/models/FullTest.java

```diff
@@ -0,0 +1,14 @@
+package com.example.nlern.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.List;
+
+@Getter @Setter
+public class FullTest {
+    Test test;
```

Comment:
So what exactly is a Test, and why should it be separate from FullTest?

Reply:
FullTest is an object that I send and receive from the frontend. It has fields that are not stored in the database, such as the list of objects deleted/added in frontend

Reply:
fullTest is a class in which, in addition to the test object and questions objects, added lists of elements that were created/removed by the frontend. it's easier to receive and send everything in one object